app/views/addons/edit.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="content">
+  <section class="page-content container-fluid">
+    <%= link_to addons_path,class:"btn btn-secondary col-md-1" do %>
+      <span>Back</span>
+    <% end %></br>

```

Comment:
Indentations!
</br> Should be on next line.

Reply:
done